go.mod

```diff
@@ -67,26 +67,26 @@ require (
 	github.com/stacklok/trusty-sdk-go v0.2.1
 	github.com/stretchr/testify v1.9.0
 	github.com/styrainc/regal v0.27.0
-	github.com/thomaspoignant/go-feature-flag v1.34.0
+	github.com/thomaspoignant/go-feature-flag v1.35.0
 	github.com/xanzy/go-gitlab v0.109.0
 	github.com/xeipuuv/gojsonschema v1.2.0
 	github.com/yuin/goldmark v1.7.4
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.54.0
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.54.0
-	go.opentelemetry.io/otel v1.29.0
+	go.opentelemetry.io/otel v1.30.0
 	go.opentelemetry.io/otel/exporters/prometheus v0.51.0
 	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.29.0
-	go.opentelemetry.io/otel/sdk v1.29.0
+	go.opentelemetry.io/otel/sdk v1.30.0
```

Comment:
Note that this bumps the otel libs so we should be careful pushing this PR (I did that mistake and then had to revert in a32f58d1f438cae8d3e157f688b09b9fd89ff16a)

Reply:
Thanks for pointing it out, I'll withdraw my approval.